.travis.yml

```diff
@@ -20,6 +20,12 @@ jobs:
     before_deploy:
     - git checkout master
     - bash ./ci-scripts/setup-npm.sh
+    - rm -rf dist
+    - mkdir dist
+    - cp -r ./ui/dist/*.js dist
+    - cp -r ./ui/dist/*.js.map dist
+    - mkdir dist/src
+    - cp -r ./ui/src/** dist/src
```

Comment:
So I think I'm still a little confused about _when_ these steps were happening before we automated? Were these commands being run by hand, one at a time? Was there a script being run? It looks like most of this was happening inside of `yarn build`. I agree with separating out the docs versus ui build, but I'm wondering if we could do that inside of their build scripts - `build:ui` and `build:docs` to keep this cleaner. If we do it this way, these steps also need to happen in pre-release and release.

Reply:
The thing is that the things were happening differently in the past. Just a few days before you began to automate things we merged a PR that introduced this separation. Before that all we had to do was npm build && npm publish.

Now the commands above do more or less the same thing and I did this once manually. I did it once because there was only one release after the said PR got merged...

Reply:
I think I will make yarn build simply build a deployable build of the components. And then have another yarn script that builds everything (this is what yarn build does today) but call it something else... would that be ok?